package.json

```diff
@@ -14,17 +29,19 @@
     "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-hooks": "^4.1.2",
     "husky": "^4.2.5",
+    "jest": "^26.4.2",
     "lerna": "^3.22.0",
     "lint-staged": "^10.4.0",
     "prettier": "^2.1.2",
+    "react-test-renderer": "^16.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-formatter-pretty": "^2.1.0",
     "stylelint-no-unsupported-browser-features": "^4.0.0",
     "stylelint-selector-bem-pattern": "^2.1.0",
     "stylelint-webpack-plugin": "^2.1.0"
   },
   "scripts": {
-    "bootstrap": "f () { npm ci && lerna link && lerna bootstrap --ci --scope=@deriv/${1:-'*'} && lerna link && npm run build:travis ;}; f",
+    "bootstrap": "f () { npm ci && lerna link && lerna bootstrap --ci --hoist --strict && lerna link && npm run build:travis ;}; f",
```

Comment:
should we update readme as to what `--strict` will implicate?

Reply:
Yep. will do right away.